### ISSUE TITLE
Better video seeking functionality

### ISSUE THREAD

**Issue by kevin-highmark** [Stremio/stremio-features#564]
**Is your feature request related to a problem? Please describe.**
It's frustrating when I want to seek faster than my default video seek which I set to 30 seconds. However, for long movies sometimes I want to jump way further or back to re-watch, but I have to seek at whatever I set it to.

**Describe the solution you'd like**
I'd like to seek at different intervals like on Kodi. I have it set at 30 seconds, but I want to be able to quickly seek longer like 1, 5, 10 min to get to where I need to go.

The way Kodi works is if you fast seek, the interval increases automatically to allow you to seek faster. Some other apps if you double press seek, it gives you fast forward options instead of seeking at a specific interval.

**Describe alternatives you've considered**
See above

**Additional context**
Uses: Android TV Stremio App, Fire TV Stick 4k

**Comment by bennodxb** [comment]
Please please please.

For me, even the ability to simply change the forward and reverse skip to different durations would be enough. I would use 30s forward 10s reverse.
exoplayer has already got this functionality it just needs the options in settings.

**Comment by kevin-highmark** [comment]
You can do it in settings already though.

**Comment by bennodxb** [comment]
You cant on android tv. What platform are you using?

**Comment by kevin-highmark** [comment]
Android TV fire TV stick 4k. I see it right there in the newest version
1.6.4 under playback settings.

**Comment by bennodxb** [comment]
Weird, Ill take another look later, but i have nvidia shield and firestick 4k max and on both of those the only available option is "set skip duration" and its a single value. You cannot choose forward and back.

That said,  i didnt mean to derail your suggestion if it had the same skip style as kodi it would be brilliant.

**Comment by kevin-highmark** [comment]
Oh I misunderstood you. You're right it's only a single value for both.

**Comment by bennodxb** [comment]
Yeah, and as you know having used a better skip system, having teh same value is useless. You go forward 30 seconds, realise you went to far, go back 30 seconds and you are where you started

**Comment by sharkooon** [comment]
Would be better to use intervals, longer pressing would result in faster forward/rewind. Now i'm only getting 10s each time..

**Comment by TRtomasz** [comment]
you can change default seek time in the settings

**Comment by sharkooon** [comment]
Thats not how it supposed to be working, sometimes you need 10 sec, sometimes you have to forward 30min...
Every streaming app has this feature, so please reopen ticket.
